util/dvsim/style.css

```diff
@@ -36,8 +31,13 @@
    white-space: -o-pre-wrap;
 }
 
+.results h2 {
+  padding-top: 15px;
```

Comment:
Why not add this to h1 and h3 as well?

Reply:
actually `h1` does not exist in our current HTML file. If I add `h3`, then the distance between `h2` and `h3` will have be 30px (including `h2` padding-bottom and `h3` padding-top).
I think this is pretty confusion. Let me change a way of writing it and add some comments.

Reply:
actually I can directly add the padding from the `.results` section, thanks for the suggestion. Hope this version is more clear.